dockge/docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: dockge_dockge_1
+      APP_PORT: 5001
+
+  docker:
+    image: docker:24.0.5-dind@sha256:3c6e4dca7a63c9a32a4e00da40461ce067f255987ccc9721cf18ffa087bcd1ef
+    privileged: true
+    network_mode: host
+    stop_grace_period: 1m
+    restart: on-failure
+    environment:
+      DOCKER_ENSURE_BRIDGE: "dind0:10.32.0.1/16"
```

Comment:
What's this IP?

Reply:
This is copied from portainer. That being said, it might be interesting to see if Dockge would clash with Portainer noew

Reply:
Two host bridges with the same IP reservations and different names, portainer and dockge, it probably will crash. You could use `dind0:10.33.0.1/16`, it is not being used by any other app.

Reply:
I was wrong about it, it will not crash but I still think it would be good to have a different ip reservation range for each.